pyomo/pysp/plugins/ecksteincombettesextension.py

```diff
@@ -363,11 +364,12 @@ def compute_updates(self, ph, subproblems, scenario_solve_counts):
 
         print("Computed sub-phi values (scenario, phi, iters-since-last-incorporated):")
         for sub_phi in sorted(sub_phi_to_scenario_map.keys()):
-            print("  %16e: " % sub_phi),
+            print("  %16e: " % sub_phi, end="")
```

Comment:
This is not going to work for Python 2. You can probably use `six.print_` instead.

Reply:
Thanks for pointing this out - I've checked to use six.print_.